tests/trestle/conftest.py

```diff
@@ -13,13 +13,85 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Common fixtures."""
+import pathlib
+import random
+import string
+from uuid import uuid4
+
 import pytest
 
-TEST_CONFIG = {}
-TEST_CONFIG['tmp_dir'] = './tmp_dir'
+from tests import test_utils
+
+from trestle.oscal.target import TargetDefinition
+from trestle.utils import fs
+
+TEST_CONFIG: dict = {}
+
+
+@pytest.fixture(scope='function')
```

Comment:
Hmm...probably I can call this `test_tmp_dir` instead of `tmp_dir` to avoid confusion. This custom `tmp_dir` lets me quickly check and debug code from IDE as it tries to create file contents. If I use the default `tmpdir`, I need to go out of my IDE and find the tmp file to inspect or debug.